processes/release_processes/scripts/start_publish_postgres_docker.sh

```diff
@@ -0,0 +1,17 @@
+# This script will launch a Postgres 11.3 container with parameters matching the expectations of Cromwell's CI configuration
```

Comment:
The fact that this is in `release_processes` with the `publish` label is sort of an artifact - we happen to use the for-Cromwell-publishing MySQL instance for development, because it's there. This probably makes more sense in a reasonable place in `scripts/` (as would the MySQL files... but that seems out of scope for this PR).